addon/src/-internal/build-registry.ts

```diff
@@ -2,18 +2,15 @@ import type { Resolver } from '@ember/owner';
 import ApplicationInstance from '@ember/application/instance';
 import Application from '@ember/application';
 import EmberObject from '@ember/object';
-
-import Ember from 'ember';
+import { Registry } from '@ember/-internals/container';
```

Comment:
I know we specified these with the barrel deprecation RFC, but I wonder if there is anything in ember-source testing to make sure we don't break such import paths?

Reply:
they're all private API, so I think it's totally fair for ember to break them if it wishes.

@ember/test-helpers should not be using private APIs haha

(we can explore migrate off of them in a separate PR)

Reply:
They may be private but if we don't do that follow up PR all the pain will be ours.

Reply:
aye, I'm doing this now 🎉